### DISCUSSION TITLE
Add issues for new contributors

### DISCUSSION THREAD

**Discussion by In-line** [GitoxideLabs/gitoxide#129] [Q&A]

**Comment by Byron** [comment]
I know the [current contributing guide](https://github.com/Byron/gitoxide/blob/main/CONTRIBUTING.md) is quite raw yet I wonder how it can be improved to be more approachable.

In short, the [backlog](https://github.com/Byron/gitoxide/projects/1#column-14505147) has a few cards already which could b turned into an issue if you chose to pick them up. I wouldn't mind you picking a card and creating an issue or PR for it to indicate you 'picked it up' . If decent workflows emerge from these interactions they could be baked into the contributing guide to be less confusing for the next reader.

What do you think?

**Reply by In-line** [reply]
@Byron this cards are not very detailed and require some digging to get started. If we could break them into few smaller and easier tasks, new contributors (like myself) will be able to get acquainted with project faster.

**Reply by Byron** [reply]
Right now that's by design, rough cards are quickly done and can be detailed on demand. Maybe another way to go about this  would be to hear what you are interested in. There is an entire work stream related to [cloning a repository](https://github.com/Byron/gitoxide/blob/6814bd54c6a148c69e23a80e7fcf82ac3401ccb5/tasks.md#L56). This highest-value item there would be to make `git-config` access a stack of configuration files to allow hierarchical access to configuration. Right now it's able to read/handle only a file at a time. With that it would be very interesting to implement something like [`gix-config`](https://github.com/Byron/gitoxide/projects/1#card-62017440).
What do you think?
